### PR TITLE
operator: set right environment variable for 'cluster-name' in operator

### PR DESCRIPTION
**Pull request by aanm**
Fixes: 7e636567e698 ("use constants for cilium-agent flags")
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6935)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Review by vadorovsky — APPROVED** [review]

File: operator/main.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ func init() {
 	flags.Int(option.ClusterIDName, 0, "Unique identifier of the cluster")
 	option.BindEnv(option.ClusterIDName)
```
Comment (tgraf) [inline operator/main.go:89]: Do we need the same fix for `--cluster-id`?

**Review by tgraf — CHANGES_REQUESTED** [review]

File: operator/main.go (hunk 1 above)
Comment (aanm) [reply operator/main.go:89]: No, the problem is that the env variable set was `CILIUM_CILIUM_CLUSTER_NAME` by the `option.BindEnv` function

**Review by tgraf — APPROVED** [review]

**Pull request merged**